### PR TITLE
Add MNIST MultiprocessParallelUpdater example

### PR DESCRIPTION
**Pull request by emcastillo**
In some cases a simpler example other than imagenet can be useful for debugging the MultiprocessParallelUpdater

### CODE REVIEW

**Comment by emcastillo** [comment]
Needs #7511 to finish correctly

File: examples/mnist/train_mnist_data_parallel_updater.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                        help='Resume the training from snapshot')
+    parser.add_argument('--unit', '-u', type=int, default=1000,
+                        help='Number of units')
+    parser.add_argument('--devices', '-d', type=str, nargs='*',
+                        default=['0', '1', '2', '3'],
+                        help='Device specifiers. Either ChainerX device '
+                        'specifiers or integers. If non-negative integer, '
+                        'CuPy arrays with specified device id are used. If '
+                        'negative integer, NumPy arrays are used')
+    parser.add_argument('--ljob', '-j', type=int,
```
Comment (Crissman) [inline examples/mnist/train_mnist_data_parallel_updater.py]: Better to set a default value.

```suggestion
    parser.add_argument('--ljob', '-j', type=int, default=4,
```

**Comment by Crissman** [comment]
imosCI, test this please.

**Comment by Crissman** [comment]
imosCI, test this please.

**Pull request merged**